### ISSUE TITLE
Latest Version of Sonarqube is not downloaded

### ISSUE THREAD

**Issue by BChip** [vanderby/SonarQube-AzureAppService#37]
Assuming the user has:
SonarQubeVersion=Latest
SonarQubeEdition=Community

I noticed in Deploy-SonarQubeAzureAppService.ps1:
```
$sortedZipFiles = $zipFiles | Sort-Object -Property @{ Expression = { [Version]([RegEx]::Match($_.href, '\d+.\d+.?(\d+)?.zip').Value -Split ".zip")[0] } }
        $latestFile = $sortedZipFiles[-1]
```

$latestFile will turn out to be sonarqube-8.0.zip, which is incorrect. The latest version is: sonarqube-8.2.0.32929.zip.

Is it safe to say if you comment-out/remove the code above? Then change to:

```
$latestFile = $zipFiles[-1]
```

I believe it is safe to assume the webpage will always have the latest version at the bottom/end; hence there is no need to perform a sort on the zip files and grab the last item.

**Comment by BChip** [comment]
Output of changes made:

![image](https://user-images.githubusercontent.com/10345051/80032066-9d87e800-84b8-11ea-952c-bd79706a75d4.png)

**Comment by vanderby** [comment]
Hey. You're referncing old code. This issue is a duplicate of #33 and fixed in #34.

The regex was updated to  '\d+.\d+.?(\d+)?.?(\d+)?.zip' to account for the fourth digit. I decided to go this route instead of just taking the last item because sometimes SQ will publish a -rc1, -rc2, -alpha release and its the last item in the list.
